content/developer/writing_plugins/creating-your-own-plugin.md

```diff
@@ -290,10 +276,10 @@ def start(**kwargs):
 ### Error Details
 
 When an operation fails due to an exception being generated (intentionally or unintentionally), the exception details are stored in the
-task_failed/task_reschduled events.
+`task_failed`/`task_rescheduled` events.
```

Comment:
Where is that?

Reply:
I don't know, and honestly, I think this sentence is unnecessary. I'll remove it.